walkthroughs/eks-app-mesh-multi-account/cleanup.sh

```diff
@@ -0,0 +1,100 @@
+#!/bin/bash
+
```

Comment:
We can use AppMesh controller to do cleanup for us. "Kubectl delete" on all the K8S resources we created might be straightforward. Controller will delete corresponding resources it created in AppMesh control plane and CloudMap as well. On our other walkthroughs, we redirect everything we create on a K8S cluster in to one manifest.yaml file and kubectl delete on that file will cleanup all the resources. I am alright with this approach as well but since we are depending on the controller to create all the resources might be better if we use it to clean them up as well.